Removing old blogs (#9)
prady0t/My-Blog: content/posts/GSoC-week-6.md

---

title: "GSoC week 11 and 12"

date: 2024-09-12

  

---

![GSoC Banner](/GSoC_Banner.png)
<img src = "/GSoC_Banner.png">

> This blog will cover the ninth and tenth week as a GSoC student.

Reviewer: ```suggestion
This blog will cover the eleventh and twelth week as a GSoC student.
```

### Work so far and for the coming weeks.

- Migrating tests to pytest part 6 : This PR covered the part 6 of pytest migration. This PR covers a lot of changes to the codebase, leveraging the power of pytest's fixtures and parameterization. With this we moved 18 files to pytest. Follow the [PR](https://github.com/pybamm-team/PyBaMM/pull/4354) to know more.

> - Migrating tests to pytest part 7 : This is the last PR for pytest migration. Almost all the files were moved to pytest. The ones remaning will be covered in a last PR. To know more follow the [PR](https://github.com/pybamm-team/PyBaMM/pull/4431).

Reviewer: ```suggestion
- Migrating tests to pytest part 7 : This is the last PR for pytest migration. Almost all the files were moved to pytest. The ones remaining will be covered in a last PR. To know more follow the [PR](https://github.com/pybamm-team/PyBaMM/pull/4431).
```

> - Final pytest migration : This PR covers all the remaining file left from the previous work. With this PR we I also conclude the main aim of the project. You can follow the [PR](https://github.com/pybamm-team/PyBaMM/pull/4443) to know more.

Reviewer: ```suggestion
- Final pytest migration : This PR covers all the remaining file left from the previous work. With this PR we also conclude the main aim of the project. You can follow the [PR](https://github.com/pybamm-team/PyBaMM/pull/4443) to know more.
```

Should be either "we" or "I"
